### ISSUE TITLE
Labels are not displayed

### ISSUE THREAD

**Issue by miggat** [JohnySeven/TWatchSK#97]
I'm not sure if this project is abandoned, but I wanted to use my Twatch again this week, formatted it and added new views using the Twach SK Designer 1.2.

After clicking on "Download DynamicViews" and watiting for the restart, the views are there, but I can only see a the button and the switch.

![image](https://user-images.githubusercontent.com/4322303/230938213-29da43bb-fd3e-4bc2-859a-9a03062ff1be.png)

However, the 2 labels doesn't appear on the watch. The space they should take is there though, but it's completely empty,

**Comment by JohnySeven** [comment]
Could you share your json file? Labels are working, maybe there is some issue with settings of those labels :-) I see you're C# guy, am I correct?

**Comment by miggat** [comment]
Yup, I work mainly with C#, don't know how you knew that :O

Here's the json for this test screen:

`{
  "views": [
    {
      "name": "New view",
      "layout": "column_mid",
      "components": [
        {
          "text": "TWS",
          "font": "montserrat14",
          "type": "label",
          "size": [
            100,
            40
          ]
        },
        {
          "text": "--",
          "font": "montserrat28",
          "binding": {
            "path": "environment.wind.speedOverGround",
            "multiply": 1.944,
            "offset": 0.0,
            "decimals": 0,
            "period": 1000
          },
          "type": "label",
          "size": [
            100,
            40
          ]
        },
        {
          "text": "test",
          "font": "montserrat14",
          "put": {
            "path": "test.number",
            "value": 1
          },
          "type": "button",
          "size": [
            100,
            40
          ]
        },
        {
          "type": "switch",
          "size": [
            60,
            40
          ]
        }
      ],
      "type": "normal",
      "background": "white",
      "size": [
        0,
        0
      ]
    }
  ]
}`

I must add than when I first tested this (at that time there were only labels, no gauge, button or switch), it used to work perfectly fine.

Oh, and also I can see data is being received on the console:

```
(141462) WS: WEBSOCKET_EVENT_DATA
(141462) WS: Received opcode=1
(141462) WS: Got SK value update environment.wind.speedOverGround
(141472) WS: Got message Delta from websocket with len=207
(141472) WS: Total payload length=207, data_len=207, current payload offset=0
```

**Comment by JohnySeven** [comment]
UI loading of Label is coded here: https://github.com/JohnySeven/TWatchSK/blob/2fa0bb16cc6f7fb88259baf8dfabc10793921225/src/ui/dynamic_label.cpp#L20 the first Label with TWS should be visible as there is no binding defined. I will load you json into watch and let you know.

**Comment by JohnySeven** [comment]
> Yup, I work mainly with C#, don't know how you knew that :O

Yeah, because you have Xamarin Forms fork you are mobile developer! Non C# guy wouldn't fork Xamarin Forms!

Also checkout my new - hot project https://github.com/JohnySeven/SKipper it's KIP replacement I'm working on done in C# and Avalonia UI - runs everywhere!

**Comment by miggat** [comment]
> Yeah, because you have Xamarin Forms fork you are mobile developer! Non C# guy wouldn't fork Xamarin Forms!
> 
> Also checkout my new - hot project https://github.com/JohnySeven/SKipper it's KIP replacement I'm working on done in C# and Avalonia UI - runs everywhere!

haha yes, that was from a veeery old project :P 

I made a quick look to SKipper and liking it so far. I put it a star so I follow it's evolution.

**Comment by JohnySeven** [comment]
> I made a quick look to SKipper and liking it so far. I put it a star so I follow it's evolution.
Thanks! Also checkout screenshots on my Mastodon account: https://social.dytrych.cloud/@jan/tagged/Skipper they will give you better look than readme and source code :-)
